Updated images on docs for authenticate azure and abbout connections (#6391)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/git/authenticate-azure.md

Connect your dbt Cloud profile to Azure DevOps using OAuth:

> 1. Click your name at the bottom left of your account.

Reviewer: Same feedback here as above

> 1. Click your name at the bottom left of your account.

Author: ```suggestion
Click your account name at the bottom of the left-side menu and click **Account settings**
```

> Click your account name at the bottom of the left-side menu and click **Account settings**

Author: ```suggestion
1. Click your account name at the bottom of the left-side menu and click **Account settings**
```

> 2. Select **Your profile** from the menu.

Author: ```suggestion
2. Scroll down to **Your profile** from the menu and select **Personal profile**.
```

> 2. Scroll down to **Your profile** from the menu and select **Personal profile**.

Author: ```suggestion
2. Scroll down to **Your profile** and select **Personal profile**.
```

> 3. Go to the Linked accounts section in the middle of the page.

Reviewer: ```suggestion
3. Go to the **Linked accounts** section in the middle of the page.
```
